tests/model_hub_tests/models_hub_common/test_convert_model.py

```diff
@@ -98,10 +98,10 @@ def _run(self, model_name, model_link, ie_device):
         inputs = self.prepare_inputs(inputs_info)
         print("Convert the model into ov::Model")
         ov_model = self.convert_model(fw_model)
-        print("Infer the original model")
-        fw_outputs = self.infer_fw_model(fw_model, inputs)
         print("Infer ov::Model")
         ov_outputs = self.infer_ov_model(ov_model, inputs, ie_device)
+        print("Infer the original model")
+        fw_outputs = self.infer_fw_model(fw_model, inputs)
```

Comment:
let us do not change it and switch off sharing mode when needed

Reply:
As agreed moved inference of original model after OV inference and added information to docs.